config/site_settings.yml

```diff
@@ -758,6 +758,11 @@ files:
     default: 'jpg|jpeg|png|gif'
     refresh: true
     type: list
+  additional_authorized_extensions_for_staff:
```

Comment:
Maybe we should just name it `authorized_extensions_for_staff`? `additional_authorized_extensions_for_staff` seems abit verbose to me. 

cc @coding-horror